okapi-core/src/main/java/org/folio/okapi/service/impl/DockerModuleHandle.java

```diff
@@ -445,7 +440,7 @@ private int getExposedPort(JsonObject b) {
         .compose(
             x -> deleteContainer(),
             // if stopContainer fails with e run deleteContainer but return original failure e
-            e -> deleteContainer().onComplete(x -> Future.failedFuture(e)))
+            e -> deleteContainer())
```

Comment:
The comment and the code no longer match.

Reply:
According to my editor the Failure.future value is lost, which was not the intention.

How about
```java
        e -> deleteContainer().eventually(x -> Future.failedFuture(e)))
```

Reply:
eventually is correct.